spec/javascripts/govuk-component/feedback-spec.js

```diff
@@ -0,0 +1,705 @@
+describe("Feedback component", function () {
+   var FIXTURE =
+    '<div class="pub-c-feedback">' +
+      '<div class="pub-c-feedback__prompt js-prompt" tabindex="-1">' +
+        '<div class="js-prompt-questions">' +
+          '<h3 class="pub-c-feedback__is-useful-question">Is this page useful?</h3>' +
+          '<a href="/contact/govuk" class="pub-c-feedback__prompt-link pub-c-feedback__prompt-link--useful js-page-is-useful" data-track-category="manualFeedbackForm" data-track-action="ffYesClick">Yes <span class="visually-hidden">this page is useful</span></a>' +
```

Comment:
In the test fixture you should be able to omit any markup/classes that are not required for js hooks.